### PR TITLE
Fix progress bar

### PR DESCRIPTION
**Pull request by m2-farzan**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -30,13 +30,14 @@ def readme():
             "Rit = Rit.Rit:main",
         ],
         "gui_scripts": [
-            # todo
+            "Rit-gui = Rit.gui:main",
```
Comment (m2-farzan) [inline setup.py:33]: Now you can launch gui window by `Rit-gui` command.

Todo: Make gui optional, as PyQt5 is a massive dependency for this project.

Comment (mo1ein) [reply setup.py:33]: What is ```from Rit.Rit import Rename``` in line 13 ?

Comment (m2-farzan) [reply setup.py:33]: It just imports `Rename` class from Rit.py in the Rit package :)

Comment (mo1ein) [reply setup.py:33]: Yeah but all files in ```Rit``` directory , I fix it :))

**Pull request merged**